study.md

```diff
@@ -68,3 +68,57 @@ entity)?
 ```md
 <!-- your response here -->
 ```
+
+## Rails Console
+
+For the subsequent questions, assume that we are in the Rails console (by
+entering `rails console` in the terminal and that the model names follow Rails
```

Comment:
I just got a note on a different repo about using `bin/rake rails dbconsole` over `rails console`. I'm going to make that change now to 016/master before the merge, but I just wanted you to know the preference.